start-server-and-test.js

```diff
@@ -0,0 +1,102 @@
+const { spawn } = require('child_process');
```

Comment:
I think this file could be named as "startVisualTests" or "runVisualTests", and running the visual tests imply starting the dev server. What do you think?

Reply:
Sure, seems much better 👍🏻